lms/djangoapps/certificates/views/support.py

```diff
@@ -208,6 +208,17 @@ def regenerate_certificate_for_user(request):
         )
         return HttpResponseServerError(_("An unexpected error occurred while regenerating certificates."))
 
+    try:
+        # Fetch CertificateInvalidation object
+        certificate_invalidation = CertificateInvalidation.objects.get(
+            generated_certificate=certificate,
+            active=True
+        )
+        # Deactivate certificate invalidation if it was fetched successfully.
+        certificate_invalidation.deactivate()
+    except:  # pylint: disable=bare-except
+        pass
```

Comment:
Its a support view for re-generation.  And I am just checking if invalidation entry exists just de-activate the record otherwise do nothing. I don't we need to log any exception or any thing.